src/avoid-autoplay.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function AudioComponent() {
+    return (
+        <div>
+            {/* Compliant */}
+            <audio preload="none" > <track kind="captions"/></audio>
+            {/* Non-compliant */}
+            <video autoPlay ><track kind="captions"/> </video>
```

Comment:
I think that you can put the attribute in lowercase
```suggestion
            <video autoplay><track kind="captions"/> </video>
```